plugins/psv-extractor/src/psv.extractor.ts

```diff
@@ -8,45 +8,72 @@ import Papa, { ParseResult } from 'papaparse'
 
 export class PsvExtractor extends AbstractExtractor {
   private readonly _options: {
-    //add if needed
+    delimiter?: ',' | '\t' | '|' | ';'
+    dynamicTyping?: boolean
+    header?: boolean
```

Comment:
⛳ I know this is exactly what `papaparse` uses but it's confusing. Meaning "what about the header are you doing?" You don't know unless you pull up the docs. So maybe we could use something like `includeHeader` or anything else you might think that would make this clear to the end user who has no idea what setting this to `true` means to the underlying library.